Gruntfile.js

```diff
@@ -61,8 +61,9 @@ module.exports = function(grunt) {
             },
             site: {
                 src: [
-                    'node_modules/d3/d3.js',
                     'node_modules/css-layout/dist/css-layout.js',
+                    'node_modules/d3/d3.js',
+                    'node_modules/svg-innerhtml/svg-innerhtml.js',
```

Comment:
That's a useful shim!

I do wonder whether we should include css-layout and svg-innerhtml in our bundle? While the current approach is good for browserify / npm users, it is not do good for people who do not have a build.

svg-innerhtml isn't on cdnjs.